### PR TITLE
ref(processing): Minor refactor

### PR DESCRIPTION
**Pull request by armenzg**
Improves the code a bit for better readability.

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -462,9 +462,6 @@ def save_error_events(
 
         is_reprocessed = is_reprocessed_event(job["data"])
 
-        # This metric can be used to track how many error events there are per platform
-        metrics.incr("save_event.error", tags={"platform": job["event"].platform or "unknown"})
```
Comment (lobsterkatie) [inline src/sentry/event_manager.py:466]: Just copying your comment from [the previous version of this PR](https://github.com/getsentry/sentry/pull/53348) over for future folks' edification:

> We now have this metric instead:
> https://github.com/getsentry/sentry/blob/c2854dc5ed7ef766b71b263aa70a7b63fdb28933/src/sentry/event_manager.py#L444

**Review by lobsterkatie — APPROVED** [review]

File: src/sentry/stacktraces/processing.py [hunk 2]
```diff
@@ -250,22 +265,21 @@ def _normalize_in_app(stacktrace: list[dict[str, str]]) -> None:
             set_in_app(frame, False)
 
 
-def normalize_stacktraces_for_grouping(data: Any, grouping_config: Any = None) -> None:
+def normalize_stacktraces_for_grouping(data, grouping_config=None) -> None:
```
Comment (armenzg) [inline src/sentry/stacktraces/processing.py:268]: Removing `Any` since it is ambiguos.

File: Makefile [hunk 3]
```diff
@@ -95,7 +95,7 @@ fetch-release-registry:
 
 run-acceptance:
 	@echo "--> Running acceptance tests"
-	pytest tests/acceptance --cov . --cov-report="xml:.artifacts/acceptance.coverage.xml"
+	pytest -s -vv tests/acceptance --cov . --cov-report="xml:.artifacts/acceptance.coverage.xml"
```
Comment (lobsterkatie) [inline Makefile:98]: Did you mean this to be part of this PR?

File: src/sentry/apidocs/extensions.py [hunk 4]
```diff
@@ -43,7 +43,7 @@ class SentryResponseSerializerExtension(OpenApiSerializerExtension):
     target_class = "sentry.api.serializers.base.Serializer"
     match_subclasses = True
 
-    def get_name(self, auto_schema: AutoSchema, direction: Direction) -> Optional[str]:
+    def get_name(self) -> Optional[str]:
```
Comment (lobsterkatie) [inline src/sentry/apidocs/extensions.py:46]: Same question with the changes in this file.

File: Makefile (hunk 3 above)
Comment (lobsterkatie) [reply Makefile:98]: Never mind - I see that you've just got them there for debugging purposes.

File: src/sentry/apidocs/extensions.py (hunk 4 above)
Comment (lobsterkatie) [reply src/sentry/apidocs/extensions.py:46]: And same realization on my part. Disregard.

**Pull request closed without merging**